test/e2e/upgrades/apps/deployments.go

```diff
@@ -46,9 +46,9 @@ func (DeploymentUpgradeTest) Name() string { return "[sig-apps] deployment-upgra
 
 func (DeploymentUpgradeTest) Skip(upgCtx upgrades.UpgradeContext) bool {
 	// The Deployment upgrade test currently relies on implementation details to probe the
-	// ReplicaSets belonging to a Deployment. As of 1.7, the client code we call into no
-	// longer supports talking to a server <1.6. (see #47685)
-	minVersion := version.MustParseSemantic("v1.6.0")
+	// ReplicaSets belonging to a Deployment. As of 1.9, the client code we call into no
+	// longer supports talking to a server <1.8. (see #47685)
+	minVersion := version.MustParseSemantic("v1.8.0")
```

Comment:
This should instead be v1.7. You don't expect <1.7 server to work against a 1.8 client.

Reply:
oic, we are still targeting 1.8 for this PR.